articles/api-auth/dynamic-client-registration.md

```diff
@@ -19,21 +19,15 @@ Dynamic Client Registration enables you to register [third-party applications](/
 
 This feature is based on the [OpenID Connect Dynamic Client Registration specification](https://openid.net/specs/openid-connect-registration-1_0.html) and in this article we will see how you can enable and use it.
 
-## Enable dynamic registration
-
-By default, the feature is disabled for all tenants. To change this, you have to:
+<%= include('../_includes/_enable-third-party-apps-info') %>
```

Comment:
I don't see the include working in Heroku,not sure if that's expected.

Reply:
fixed